packages/fixtures/README.md

```diff
@@ -0,0 +1,75 @@
+# Slack Fixtures
+
+This package contains type-safe fixtures for testing slack applications.
+
+## Install
+
+```bash
+yarn add --dev @slack-wrench/fixtures
+# or
+npm install --save-dev @slack-wrench/fixtures
+```
+
+## Usage
+
+Currently, we only support a subset of Slack's [Event's API](https://api.slack.com/events-api). We're implementing them as we need them and hope to have others do the same.
+
+### Events
+
+```typescript
+import { events } from '@slack-wrench/fixtures';
+```
+
+### Slash Commands
+
+```
+events.slashCommand(command, [options])
```

Comment:
what standard are you using for documenting this interface? this makes it look like you pass in options inside an array?

docs here might be better as 

```ts
events.slashCommand(command, options?)
```

? also I've seen this before as 

```ts
events.slashCommand(command [, options])
```

Reply:
could just steal the typescript signature here? then you don't have to duplicate the types/optionality below.
```ts
events.slashCommand: (command: string, options?: Partial<SlashCommand>) => SlashCommand
```

Reply:
I was aiming for the second one, there isn't really a standard? I'll lean more to typescript.

Reply:
Changing to typescript syntax. Javascripters who don't know typescript I've found are often afraid of TS syntax. 

I was trying to find something in the middle. Honestly this should all be generated. Probably that will be the way to fix the confusion one way or another. Till then, this will do.